main/postgres/initdb.d/README.markdown

```diff
@@ -0,0 +1,14 @@
+This directory helps you prepare your Postgres instance for MPortal to run.
```

Comment:
To be clear this directory was created to initialize the Docker Postgresql instance from scratch. It was not originally intended as documentation, but does double duty in that sense.